### PR TITLE
update pyfaidx/gffutils

### PR DESCRIPTION
**Pull request by mschubert**
* dev-python/pyfaidx: version bump 0.8.1.1 + move from sci-biology to dev-python
* sci-bioloy/gffutils: version bump 0.13; fix tests
* dev-python/pybedtools: version bump 0.10.0 + move from sci-biology to dev-python

### CODE REVIEW

**Comment by AndrewAmmerlaan** [comment]
This needs a `move` entry in `profiles/updates`.

**Comment by mschubert** [comment]
I also added an update to pybedtools to this PR (used to revdep on gffutils, not anymore)

File: dev-python/pybedtools/pybedtools-0.10.0.ebuild [hunk 1]
```diff
@@ -24,8 +24,9 @@ RDEPEND="
 	dev-python/pandas[${PYTHON_USEDEP}]
 	dev-python/matplotlib[${PYTHON_USEDEP}]
 "
-
-BDEPEND="dev-python/cython[${PYTHON_USEDEP}]"
+DEPEND="
+	dev-python/cython[${PYTHON_USEDEP}]
+"
```
Comment (AndrewAmmerlaan) [inline dev-python/pybedtools/pybedtools-0.10.0.ebuild:29]: This is not necessary, cython dep is added by `DISTUTILS_EXT=1` (and it is definitely a bdep not dep)

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**